README.md

```diff
@@ -4,6 +4,14 @@ This tool converts URDF files into Webots PROTO files.
 
 ## Install
 
+### From pip
+
+```
+pip install urdf2webots
+```
+
```

Comment:
On macOS, a pip binary cannot be run as-is, as explained here: https://stackoverflow.com/a/43368894/2210777

```suggestion

On macOS, export the pip binary path to the PATH: `export PATH="/Users/$USER/Library/Python/3.7/bin:$PATH"`

```